drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -120,6 +120,7 @@ public void setChromeOptions(WebDriverConfiguration configuration, DesiredCapabi
         String browser = configuration.getBrowser().toLowerCase();
         if (browser.equals("chromeheadless")) {
             chromeOptions.addArguments("--headless");
+            chromeOptions.addArguments("--window-size=1200,600");
```

Comment:
I know that the `dimension` doesn't work for this case - because there is nothing implemented like this... 
What I meant is adding the support for it - that when somebody uses chromeHeadless browser and specifies dimensions, then the `--window-size` argument should be set automatically.